src/App.tsx

```diff
@@ -1,8 +1,27 @@
 import './styles/App.css';
 import Header from './ui/Header';
+import * as React from 'react';
+import { BrowserRouter as Router, Route, Routes, Navigate } from 'react-router-dom';
+import YourRecipe from './pages/YourRecipe';
+import Recipes from './pages/Recipes';
+import About from './pages/About';
+import Admin from './pages/Admin';
+import { Nav } from './ui/Nav';
 
 function App() {
-  return <Header></Header>;
+  return (
+    <Router basename="/oslo-kokebook/">
+      <Header />
+      <Nav />
+      <Routes>
+        <Route exact path="/" element={<Navigate replace to="/your-recipe" />} />
```

Comment:
my ide flagged the exact prop, looks like it can be removed since we're using v6 https://stackoverflow.com/a/69866593/13777199